tox.ini

```diff
@@ -20,20 +20,26 @@ basepython =
 setenv =
     PURE_PYTHON = 1
 
+[testenv:py37-pure]
+basepython =
+    python3.7
+setenv =
+    PURE_PYTHON = 1
```

Comment:
Maybe we want to add this to the Travis build matrix as well?

Reply:
Or does the PyPy3 branch effectively cover that?